### DISCUSSION TITLE
Support for multi-core CPUs?

### DISCUSSION THREAD

**Discussion by bigscientist** [LuxDL/Lux.jl#775] [Q&A] [answered]
I'm using Lux and Diffeqflux  to train neural ODEs, and I have a intel core12500H cpu. 
So how to apply "EnsembleProblem" in OrdinaryDiffEq to utilze multi-CPUs?

**Comment by avik-pal** [comment, marked as answer]
Batching a neural network in a multi core setup is different from how you batch an ode. All processes use multiple cores by default and don't need any special ensemble problem setup.
